defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 openwisp2_python: python3
 ansible_python_interpreter: /usr/bin/python3
 openwisp2_network_topology: false
-openwisp2_controller_version: "0.7.post1"
+openwisp2_controller_version: "0.8.0"
```

Comment:
Thanks for updating the PR @nemesisdesign but it will not make much difference due to [compatibility versioning](https://www.python.org/dev/peps/pep-0440/#compatible-release). 

I think both `~=0.8.0` and ~=`0.8.1` translates to `~=0.8.*`